libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -1992,15 +2009,19 @@ void WaveTrack::Disjoin(double t0, double t1)
       // sequence is longer than the minimum number, split-delete the region
 
       sampleCount seqStart = -1;
-      auto start = clip->TimeToSamples(std::max(.0, t0 - startTime));
-      auto end = clip->TimeToSamples(std::min(endTime, t1) - startTime);
+      auto start = interval->TimeToSamples(std::max(.0, t0 - startTime));
+      auto end = interval->TimeToSamples(std::min(endTime, t1) - startTime);
 
       auto len = (end - start);
       for (decltype(len) done = 0; done < len; done += maxAtOnce) {
          auto numSamples = limitSampleBufferSize(maxAtOnce, len - done);
 
-         clip
-            ->GetSamples(buffers.data(), floatSample, start + done, numSamples);
+         auto bufferIt = buffers.begin();
+
+         for (auto channel : interval->Channels())
+            channel->GetSamples(
+               *bufferIt++, floatSample, start + done, numSamples);
```

Comment:
This works.

Maybe `WaveTrack::Interval` should instead implement `WideSampleSequence` and reduce that to a single call.

Not in this PR.